src/Umbraco.Web.UI.Client/src/views/propertyeditors/rte/rte.prevalues.controller.js

```diff
@@ -65,9 +110,11 @@ angular.module("umbraco").controller("Umbraco.PrevalueEditors.RteController",
             
         });
 
-        //when the scope is destroyed we need to unsubscribe
+        // when the scope is destroyed we need to unsubscribe
         $scope.$on('$destroy', function () {
             unsubscribe();
         });
 
-    });
+        // load TinyMCE skin which contains css for font-icons
+        assetsService.loadCss("lib/tinymce/skins/umbraco/skin.min.css");
```

Comment:
I just load and re-use the ```skin.min.css``` ... I could pull only the part for icons in a separate css, but probably not many (kilo)bytes that are saved and when re-using this we don't have to maintain it in other files (ensure classes for icons etc. are identical).